### PR TITLE
Modified the README.md file

### PR DESCRIPTION
**Pull request by HalynaDumych**
Fixed bugs that were found in the generated Doxygen documentation
and modified the content to align it with the Developer's Guide.

Relates-to: OLPEDGE-1242

Signed-off-by: Halyna Dumych <ext-halyna.dumych@here.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,154 +30,151 @@ The HERE OLP SDK for C++ is a C++ client for [HERE Open Location Platform](https
 
 ## Why Use
 
-The HERE OLP SDK for C++ provides support for core HERE Open Location Platform (OLP) use cases through a set of native C++ interfaces. The SDK is intended to save your time and effort on using OLP REST APIs and provides a set of stable APIs that simplify complex OLP operations and remove the burden of keeping up to date with the latest OLP REST API changes.
+The Open Location Platform (OLP) SDK for C++ provides support for the core HERE OLP use cases through a set of native C++ interfaces. The SDK is intended to save your time and effort on using OLP REST APIs. It provides a set of stable APIs that simplify complex OLP operations and keeps up to date with the latest OLP REST API changes.
 
-It's a modern (C++11 or later), lightweight, and modular SDK with minimal dependencies that is targeted towards a wide range of hardware platforms, from embedded devices to desktops.
+The OLP SDK for C++ is a modern (C++ 11), lightweight, and modular SDK with minimal dependencies targeted towards a wide range of hardware platforms from embedded devices to desktops.
```
Comment (mykolamalik) [inline README.md]:
```suggestion
The OLP SDK for C++ is a modern (C++11), lightweight, and modular SDK with minimal dependencies targeted towards a wide range of hardware platforms from embedded devices to desktops.
```

File: README.md [hunk 2]
```diff
@@ -30,154 +30,151 @@ The HERE OLP SDK for C++ is a C++ client for [HERE Open Location Platform](https
...
-cmake -DOLP_SDK_BUILD_DOC=ON ..
-cmake --build . --target docs
-```
+To install the SDK on your system:
 
-#### Install
+1. Install all the dependencies needed for the SDK.<br>For more information on dependencies, see the [Dependencies](#dependencies) and [Additional Linux Dependencies](#additional-linux-dependencies) sections.
 
-By default this SDK downloads and compiles its dependencies. The versions of downloaded dependencies may conflict with versions already installed on your system, so they are not added to the install targets.
+2. (Optional) To find the required dependencies in the system, set the  `OLP_SDK_BUILD_EXTERNAL_DEPS` flag to `OFF`.
```
Comment (mykolamalik) [inline README.md]:
```suggestion
2. (Optional) To find the required dependencies in the system, set the `OLP_SDK_BUILD_EXTERNAL_DEPS` flag to `OFF`.
```

File: README.md [hunk 3]
```diff
@@ -30,154 +30,151 @@ The HERE OLP SDK for C++ is a C++ client for [HERE Open Location Platform](https
...
-> **Note:**
-> Visual Studio uses a default build directory that has a long path name. It's recommended that you edit the generated `CMakeSettings.json` and change the build directory to a short path name, since dependencies for the SDK are installed within the build directory. This ensures that the maximum length for each individual path is not greater than 260 characters. For details, see [here](https://docs.microsoft.com/en-us/windows/desktop/fileio/naming-a-file).
+```bash
+    mkdir build && cd build
+    cmake ..
+    cmake --build .
+```
 
-### CMake Flags
+If you cannot build the SDK on Windows using this  instruction, see [Build on Windows](#build-on-windows)
```
Comment (mykolamalik) [inline README.md]:
```suggestion
If you cannot build the SDK on Windows using this instruction, see [Build on Windows](#build-on-windows)
```

File: README.md (hunk 1 above)
Comment (HalynaDumych) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (HalynaDumych) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (HalynaDumych) [reply README.md]: Fixed

**Pull request merged**